### DISCUSSION TITLE
Can't install mkcert

### DISCUSSION THREAD

**Discussion by markonikolas** [FiloSottile/mkcert#441] [Q&A] [answered]
Hi, 
I'm having an issue with installing mkcert inside docker container on M1. It seems that it's a CPU problem. Found a workaround by generating certs on host machine and moving them inside container. The reasoning behind installing mkcert in the container is that I would like to generate certs on docker build.

<img width="690" alt="Screenshot 2021-08-31 at 14 29 26" src="https://user-images.githubusercontent.com/36996299/131629592-4c2c568a-af81-4be2-bcd2-c4de34472c24.png">

**Comment by rfay** [comment, marked as answer]
homebrew does not support arm64 on linux at all. 

You can download and install a linux/arm64 version of mkcert from https://github.com/FiloSottile/mkcert/releases
